Cargo.toml

```diff
@@ -38,6 +38,8 @@ mimalloc = { version = "0.1.38", default-features = false }
 async-once-cell = {version="0.5.3", default-features=false}
 actix-governor = {version="0.5.0", default-features=false}
 mini-moka = { version="0.10", optional = true, default-features=false, features=["sync"]}
+enumflags2 = "0.7.8"
+time ={version = "0.3.30", features = ["serde-human-readable"]}
```

Comment:
I would suggest not including this crate and instead using the Unix epoch method to compute random delay between requests